project/Dependencies.scala

```diff
@@ -42,11 +45,26 @@ object Dependencies {
 
   val coreModuleDeps = Seq(
     scalaJSDOM,
-    chandu0101Macros
+    chandu0101Macros,
+    npmDependencies in Compile += "react" -> Version.react
   )
 
-  val webModuleDeps = Seq(
+  val universalModuleDeps = Seq(
+    npmDependencies in Compile ++= Seq(
+      "react" -> "15.4.0-rc.4",
+      "react-native" -> "0.38.0"
```

Comment:
hmm i think we have a problem here , universal module contains common components between mobile and web module , web version is backed by `react-native-web` . can you move react-native dependency to mobile module ? and also can you please change scope of npmDependencies to provided ...